plugins/webp-uploads/tests/test-picture-element.php

```diff
@@ -260,9 +260,16 @@ function ( string $content ): string {
 		);
 	}
 
-	public function test_disable_responsive_image_with_picture_element(): void {
+	/**
+	 * @dataProvider data_provider_test_disable_responsive_image_with_picture_element
+	 *
+	 * @param Closure $add_filter The filter.
+	 */
+	public function test_disable_responsive_image_with_picture_element( ?Closure $add_filter ): void {
```

Comment:
There is never `null` passed in here, so this should be just `Closure`:

```suggestion
	public function test_disable_responsive_image_with_picture_element( Closure $add_filter ): void {
```

Also, instead of `$add_filter` I think `$set_up` is better.